service/controller/resource_set.go

```diff
@@ -537,6 +538,29 @@ func NewResourceSet(config ResourceSetConfig) (*controller.ResourceSet, error) {
 			return nil, microerror.Mask(err)
 		}
 
+		/*
+			_, vnet, err := net.ParseCIDR(key.VnetCIDR(cr))
+			if err != nil {
+				return nil, microerror.Mask(err)
+			}
+			subnets, err := network.Compute(*vnet)
+			if err != nil {
+				return nil, microerror.Mask(err)
+			}
+		*/
+
+		subnets := &network.Subnets{}
+
+		tenantAzureClientCredentialsConfig, err := credential.GetTenantAzureClientCredentialsConfig(config.K8sClient.K8sClient(), cr)
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
+		authorizer, err := tenantAzureClientCredentialsConfig.Authorizer()
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
+
+		subscriptionID, partnerID, err := credential.GetSubscriptionAndPartnerID(config.K8sClient.K8sClient(), cr)
```

Comment:
I think this is not needed? The next line already gets that information, and I think we don't need the `authorizer`.